### ISSUE TITLE
Problem with the RPW Interface on Windows 10

### ISSUE THREAD

**Issue by Benzix123** [mcRPW/rpw#145]
Hi, whenever I open RPW everything is small. I've tried resizing it and going into large window mode but nothing helps. I've tried reinstalling it and everything. I am running it with the latest version of Java! Please help.

**Comment by MCrafterzz** [comment]
I have Windows 10 and it works for me

**Comment by Benzix123** [comment]
I'll send a screenshot so you can see, please please help I've been trying to fix this for 1 month now...

**Comment by MCrafterzz** [comment]
Why did you remove your comment? Did it start working?

**Comment by Benzix123** [comment]
No, it didn't I guess I just thought... there was no hope for me because I have an MSI computer. It just doesn't work I've tried everything here is a screenshot.
![screenshot 7](https://user-images.githubusercontent.com/25119493/37876734-1a6f19e2-301f-11e8-9394-dc8d4731925d.png)

**Comment by Benzix123** [comment]
Sorry I could not respond sooner.

**Comment by MCrafterzz** [comment]
hmm

**Comment by Benzix123** [comment]
I am using the latest version of Java and my screen size is 15.6 inches. I don't know if anyone can fix this...

**Comment by MCrafterzz** [comment]
hmm, I think it is because of java so I have no idea how to fix this :C

**Comment by Benzix123** [comment]
I have...

**Comment by MCrafterzz** [comment]
I'm really sorry but I have no idea what I can do about this :C

**Comment by TheGreatEye** [comment]
Very late response, but what is your screen resolution? If it's more than 1080p it could be a scaling issue.
